### PR TITLE
[Bugfix] Fixed CVEs in gangway image by migrating to Debian base image

### PR DESCRIPTION
**Pull request by fanzhangio**
- Comple gangway based on go1.13-stretch
- Build gangway image on top of amd64/debian latest image
- The new built gangway image was tested

Fixe #158 
Signed-off-by: FAN ZHANG <fanz@vmware.com>

### CODE REVIEW

**Comment by fanzhangio** [comment]
@stevesloka Could you please take a look at this PR ? Thanks.

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN esc -o cmd/gangway/bindata.go templates/
 
 RUN CGO_ENABLED=0 GOOS=linux go install -ldflags="-w -s" -v github.com/heptiolabs/gangway/...
 
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
+FROM amd64/debian:latest
```
Comment (vijaykatam) [inline Dockerfile:16]: Can this be debian-slim? Suggest pinning to a specific version. For example, `10.3-slim`.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12
+FROM golang:1.13.8-stretch
```
Comment (stevesloka) [inline Dockerfile:1]: Go 1.14 is out, maybe just move to that?

File: Dockerfile (hunk 1 above)
Comment (stevesloka) [reply Dockerfile:16]: +1 pinning a specific version

File: Dockerfile (hunk 2 above)
Comment (fanzhangio) [reply Dockerfile:1]: Updated golang image  with 1.14.2-stretch

File: Dockerfile (hunk 1 above)
Comment (fanzhangio) [reply Dockerfile:16]: Updated with debian:9.12-slim as base image

**Comment by fanzhangio** [comment]
@stevesloka Could you please take a look again ? Thanks.

**Pull request merged**